"type" needs to be a reserved word

The NetBox REST API uses "type" as a method parameter name frequently:

```
.zig-cache/o/55e6838467cdb8b3a0131e987e0f97ee/api.zig:109415:3200: error: name shadows primitive 'type'
                pub fn vpn_l2vpns_list_Result(client: *Client, contact: ?[]const u8, contact__n: ?[]const u8, contact_group: ?[]const u8, contact_group__n: ?[]const u8, contact_role: ?[]const u8, contact_role__n: ?[]const u8, created: ?[]const u8, created__empty: ?[]const u8, created__gt: ?[]const u8, created__gte: ?[]const u8, created__lt: ?[]const u8, created__lte: ?[]const u8, created__n: ?[]const u8, created_by_request: ?[]const u8, description: ?[]const u8, description__empty: ?bool, description__ic: ?[]const u8, description__ie: ?[]const u8, description__iew: ?[]const u8, description__iregex: ?[]const u8, description__isw: ?[]const u8, description__n: ?[]const u8, description__nic: ?[]const u8, description__nie: ?[]const u8, description__niew: ?[]const u8, description__nisw: ?[]const u8, description__regex: ?[]const u8, export_target: ?[]const u8, export_target__n: ?[]const u8, export_target_id: ?[]const u8, export_target_id__n: ?[]const u8, id: ?[]const u8, id__empty: ?bool, id__gt: ?[]const u8, id__gte: ?[]const u8, id__lt: ?[]const u8, id__lte: ?[]const u8, id__n: ?[]const u8, identifier: ?[]const u8, identifier__empty: ?bool, identifier__gt: ?[]const u8, identifier__gte: ?[]const u8, identifier__lt: ?[]const u8, identifier__lte: ?[]const u8, identifier__n: ?[]const u8, import_target: ?[]const u8, import_target__n: ?[]const u8, import_target_id: ?[]const u8, import_target_id__n: ?[]const u8, last_updated: ?[]const u8, last_updated__empty: ?[]const u8, last_updated__gt: ?[]const u8, last_updated__gte: ?[]const u8, last_updated__lt: ?[]const u8, last_updated__lte: ?[]const u8, last_updated__n: ?[]const u8, limit: ?i64, modified_by_request: ?[]const u8, name: ?[]const u8, name__empty: ?bool, name__ic: ?[]const u8, name__ie: ?[]const u8, name__iew: ?[]const u8, name__iregex: ?[]const u8, name__isw: ?[]const u8, name__n: ?[]const u8, name__nic: ?[]const u8, name__nie: ?[]const u8, name__niew: ?[]const u8, name__nisw: ?[]const u8, name__regex: ?[]const u8, offset: ?i64, ordering: ?[]const u8, owner: ?[]const u8, owner__n: ?[]const u8, owner_group: ?[]const u8, owner_group__n: ?[]const u8, owner_group_id: ?[]const u8, owner_group_id__n: ?[]const u8, owner_id: ?[]const u8, owner_id__n: ?[]const u8, q: ?[]const u8, slug: ?[]const u8, slug__empty: ?bool, slug__ic: ?[]const u8, slug__ie: ?[]const u8, slug__iew: ?[]const u8, slug__iregex: ?[]const u8, slug__isw: ?[]const u8, slug__n: ?[]const u8, slug__nic: ?[]const u8, slug__nie: ?[]const u8, slug__niew: ?[]const u8, slug__nisw: ?[]const u8, slug__regex: ?[]const u8, status_param: ?[]const u8, status__empty: ?bool, status__ic: ?[]const u8, status__ie: ?[]const u8, status__iew: ?[]const u8, status__iregex: ?[]const u8, status__isw: ?[]const u8, status__n: ?[]const u8, status__nic: ?[]const u8, status__nie: ?[]const u8, status__niew: ?[]const u8, status__nisw: ?[]const u8, status__regex: ?[]const u8, tag: ?[]const u8, tag__n: ?[]const u8, tag_id: ?[]const u8, tag_id__n: ?[]const u8, tenant: ?[]const u8, tenant__n: ?[]const u8, tenant_group: ?[]const u8, tenant_group__n: ?[]const u8, tenant_group_id: ?[]const u8, tenant_group_id__n: ?[]const u8, tenant_id: ?[]const u8, tenant_id__n: ?[]const u8, type: ?[]const u8, type__empty: ?bool, type__ic: ?[]const u8, type__ie: ?[]const u8, type__iew: ?[]const u8, type__iregex: ?[]const u8, type__isw: ?[]const u8, type__n: ?[]const u8, type__nic: ?[]const u8, type__nie: ?[]const u8, type__niew: ?[]const u8, type__nisw: ?[]const u8, type__regex: ?[]const u8, updated_by_request: ?[]const u8) !ApiResult(PaginatedL2VPNList) {
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ^~~~
.zig-cache/o/55e6838467cdb8b3a0131e987e0f97ee/api.zig:109415:3200: note: consider using @"type" to disambiguate
```

--- a/src/generators/unified/ident_utils.zig
+++ b/src/generators/unified/ident_utils.zig
@@ -74,6 +74,7 @@ test "isReservedIdent" {
     try std.testing.expect(isReservedIdent("if"));
     try std.testing.expect(isReservedIdent("return"));
     try std.testing.expect(isReservedIdent("struct"));
+    try std.testing.expect(isReservedIdent("type"));
     try std.testing.expect(!isReservedIdent("foo"));
     try std.testing.expect(!isReservedIdent(""));
 }

--- a/src/generators/unified/ident_utils.zig
+++ b/src/generators/unified/ident_utils.zig
@@ -16,7 +16,7 @@ pub fn isReservedIdent(name: []const u8) bool {
         "extern",    "false",    "fn",        "for",    "if",          "inline",   "isize",          "linksection",
         "noalias",   "noreturn", "nosuspend", "null",   "opaque",      "or",       "orelse",         "packed",
         "pub",       "resume",   "return",    "struct", "suspend",     "switch",   "test",           "threadlocal",
-        "true",      "try",      "undefined", "union",  "unreachable", "usize",    "usingnamespace", "var",
+        "true",      "try",      "type",      "undefined", "union",  "unreachable", "usize",    "usingnamespace", "var",
         "void",      "volatile", "while",
     };
     for (reserved) |word| {

